acceptance_tests/features/pages/breadcrumbs.py

```diff
@@ -1,13 +1,16 @@
 from acceptance_tests import browser
+from selenium.common.exceptions import NoSuchElementException
 
+driver = browser.driver
```

Comment:
Why is `browser.driver` here?

Reply:
The function I'm using is only present in driver.